### PR TITLE
Update veme.c

### PR DESCRIPTION
**Pull request by benwaffle**

### CODE REVIEW

**Comment by benwaffle** [comment]
pls merge

File: src/veme.c [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #include <stdio.h>
 #include <stdlib.h>
+#include "veme.h"
```
Comment (cbouwense) [inline src/veme.c:3]: veme.h is deprecated, we should probably be using `meme.h` here

**Review by cbouwense — CHANGES_REQUESTED** [review]
let me know what you think of my comments. I think we should have a meeting about the use of veme.h

File: src/veme.c (hunk 1 above)
Comment (benwaffle) [reply src/veme.c:3]: i'll give you 4 scrum points if you ignore this

Comment (cbouwense) [reply src/veme.c:3]: no one needs to know about this

**Pull request merged**